ui/src/components/Read/index.vue

```diff
@@ -347,27 +344,25 @@ function handleCodeData() {
     getTableData()
 }
 function autoRefreshTimeRange() {
-    if (isDatePicker.value) {
```

Comment:
This variable is still useful, but the original version still has some flaws. Let me clarify the logic.

### "last 15 minutes" is selected. 
1. If user click "🔍" , both the start and end dates have been updated to reflect the current time.
2. If user click "↻", the dates remain unchanged.

### A fixed time range is selected 
1. If user click "🔍" , the dates remain unchanged.
2. If user click "↻", the dates remain unchanged.

Based on the four cases mentioned above, it appears that "autoRefreshTimeRange" feature has limited functionality and works only in one scenario.

Reply:
If a fixed time range is selected, the start and end should remain unchanged whenever you click "🔍" or "↻". That's why you have to use a variable to mark this status.

The fixed time range allows users to select specific data within a certain time range without having to change the time when clicking searching or refreshing buttons.

Reply:
Changed accordingly and upated the description. Please verify the new video demo.